exporter/otlpexporter/internal/arrow/common_test.go

```diff
@@ -70,17 +66,21 @@ type noisyTest bool
 const Noisy noisyTest = true
 const NotNoisy noisyTest = false
 
-func newTestTelemetry(t *testing.T, noisy noisyTest) component.TelemetrySettings {
+func newTestTelemetry(t *testing.T, noisy noisyTest) (_ component.TelemetrySettings, obslogs *observer.ObservedLogs) {
```

Comment:
question: Is the named return here important?

Reply:
Fixed. 

I like named return values for declaring variables, but don't like bare return statements-- so if I use them I'll make sure to leave one parameter unnamed. In this case, I moved a return statement to avoid.